### ISSUE TITLE
Will there be teset set released? Currently only train and val set canbe downloaded

### ISSUE THREAD

**Issue by ljj898** [yinanhe/ForgeryNet#3]

**Comment by yinanhe** [comment]
Hi, we are about to hold a competition, in the competition, the test set will be used as important data for evaluation. We will release part of the test set in the competition first.

**Comment by ljj898** [comment]
Thanks very much!
But only the training_list of the training set can be downloaded now,will the validation list can be released recently?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yinanhe/ForgeryNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月3日(星期六) 下午3:48
***@***.***&gt;;
抄送:&nbsp;"Rice &amp; ***@***.******@***.***&gt;;
主题:&nbsp;Re: [yinanhe/ForgeryNet] Will there be teset set released? Currently only train and val set canbe downloaded (#3)





 
Hi, we are about to hold a competition, in the competition, the test set will be used as important data for evaluation. We will release part of the test set in the competition first.
